scryfall-sdk.gemspec

```diff
@@ -12,4 +12,5 @@ Gem::Specification.new do |s|
     'https://github.com/MatthewSerre/scryfall-sdk'
   s.license = 'MIT'
   s.required_ruby_version = '>= 2.6.0'
+  s.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
Do you know what caused this? Did it just happen when you ran the commands to build and install the gem?

Reply:
This line was added by Rubocop after I enabled all Cops.